modules/stitching/include/opencv2/stitching/detail/motion_estimators.hpp

```diff
@@ -328,9 +328,18 @@ class CV_EXPORTS_W BundleAdjusterAffinePartial : public BundleAdjusterBase
 enum WaveCorrectKind
 {
     WAVE_CORRECT_HORIZ,
-    WAVE_CORRECT_VERT
+    WAVE_CORRECT_VERT,
+    WAVE_CORRECT_AUTO
 };
 
+/** @brief Tries to detect the wave correction kind depending
+on whether a panorama spans horizontally or vertically
+
+@param rmats Camera rotation matrices.
+@return The correction kind to use for this panorama
+ */
+WaveCorrectKind CV_EXPORTS_W autoDetectWaveCorrectKind(const std::vector<Mat> &rmats);
```

Comment:
> CV_EXPORTS_W

Do we need wrapping this API into Python bindings?
If no, please use just `CV_EXPORTS` (preferable on a separate line).

```
CV_EXPORTS
WaveCorrectKind autoDetectWaveCorrectKind(const std::vector<Mat> &rmats);
```

Reply:
Done, no need to wrap this